### PR TITLE
Fix validator: nominalValue is optional unless estimated!=1 anywhere

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: petab/lint.py [hunk 1]
```diff
@@ -166,6 +166,23 @@ def check_parameter_df(
...
+    non_estimated_par_ids = list(df.index[(df[ESTIMATE] != 1)
+                                          | (df[ESTIMATE] != '1')])
+    if non_estimated_par_ids:
+        if NOMINAL_VALUE not in df:
+            raise AssertionError("Parameter table contains parameters "
+                                 f"{non_estimated_par_ids} that are not "
+                                 "specified to be estimated, "
+                                 f"but column {NOMINAL_VALUE} is missing.")
+        try:
+            df.loc[non_estimated_par_ids, NOMINAL_VALUE].apply(float)
```
Comment (yannikschaelte) [inline petab/lint.py]:
```suggestion
            df.loc[non_estimated_par_ids, NOMINAL_VALUE].apply(int)
```

Comment (yannikschaelte) [reply petab/lint.py]: ist ``in [0,1]`` explicitly checked anywhere?

Comment (dweindl) [reply petab/lint.py]: yes `assert_parameter_estimate_is_boolean`

Comment (dweindl) [reply petab/lint.py]: But this one is about nominalValue, not about estimate. So float seems more appropriate.

**Pull request merged**